optimum/commands/export/neuron.py

```diff
@@ -88,19 +88,16 @@ def parse_args_neuron(parser: "ArgumentParser"):
     input_group.add_argument(
         "--batch_size",
         type=int,
-        default=1,
```

Comment:
I think, `batch_size` might be the only shape we accept a default for, since it's for inference. But do not have a strong opinion on this.

Reply:
Sounds good! Also compared to sequence length this should error when you are not providing the right batch size